### ISSUE TITLE
ERR-104: Tag Mismatch - Line 291

### ISSUE THREAD

**Issue by wangmuge** [xornorik/SVPinView#41]
----------SVPinView Error----------
ERR-104: Tag Mismatch - Line 291
-----------------------------------
2020-04-17 09:01:25.854738+0800 nookprint[88745:19458510] *** Assertion failure in -[UICollectionView _createPreparedCellForItemAtIndexPath:withLayoutAttributes:applyAttributes:isFocused:notify:], /BuildRoot/Library/Caches/com.apple.xbs/Sources/UIKitCore_Sim/UIKit-3899.22.15/UICollectionView.m:2248
2020-04-17 09:01:25.858613+0800 nookprint[88745:19458510] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'the cell returned from -collectionView:cellForItemAtIndexPath: does not have a reuseIdentifier - cells must be retrieved by calling -dequeueReusableCellWithReuseIdentifier:forIndexPath:'

**Comment by xornorik** [comment]
@wangmuge Thanks for reporting the error!
Could you confirm if this is happening consistently or if it's just a sporadic behaviour?

**Comment by snievesZafiro** [comment]
This error is when you copy the files manually and not use cocoapods

**Comment by syedrazackimran** [comment]
Hi, I am also facing the same issue. when using manually not in cocoa pods


```
----------SVPinView Error----------
ERR-104: Tag Mismatch - Line 291
-----------------------------------

*** Assertion failure in -[UICollectionView _createPreparedCellForItemAtIndexPath:withLayoutAttributes:applyAttributes:isFocused:notify:], /Library/Caches/com.apple.xbs/Sources/UIKitCore_Sim/UIKit-3920.31.102/UICollectionView.m:2331

 *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'the cell returned from -collectionView:cellForItemAtIndexPath: does not have a reuseIdentifier - cells must be retrieved by calling -dequeueReusableCellWithReuseIdentifier:forIndexPath:'
```

**Comment by xornorik** [comment]
@snievesZafiro 
Thank you for diagnosing this issue. I was able to fix it in the latest release. 

@wangmuge @syedrazackimran 
This happened because after the files were copied, the target for the SVPinField file was set to SVPinView instead of the parent target. Refer #49 

I have fixed this in **v1.0.9**
